third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -2151,6 +2193,43 @@ resource "google_container_cluster" "with_node_pool" {
 }`, cluster, nodePool)
 }
 
+func testAccContainerCluster_autoprovisioning(cluster string, autoprovisioning bool) string {
+	config := fmt.Sprintf(`
```

Comment:
Totally up to you, but I think it might be cleaner if you used %s instead of concatenating, so like:
```go
var autoscalingCfg string
if autoprovisioning {
  autoscalingCfg = [stuff]
} else {
  autoscalingCfg = [other stuff]
}

return fmt.Sprintf(`
[things]

%s

[other things]
`, autoscalingCfg)
```